public/404.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="/static/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="favicon.ico" type="image/x-icon" />
```

Comment:
I'm going to approve this PR; it's all good to go.  Just curious, why is there a change in this file?

Reply:
Actually, this was supposed to be in the error page PR. Nevertheless, the favicon had to be placed in the static directory for Django to see it, but it does not apply anymore since we have separated the frontend and backend.